README.md

```diff
@@ -525,6 +527,14 @@ Options for target 'qemu':
   --qemu/unaligned      Permit un-aligned loads/stores (default: 'False')
   --qemu/statistics     Collect statistics on out-of-bounds capability creation. (default: 'False')
 
+Options for target 'FreeRTOS':
+  --(run-)freertos/demo Demo  Build/run a specific FreeRTOS Demo. Currently, available Demos are RISC-V_Galois_P1
+                        and RISC-V-Generic (default: 'RISC-V-Generic').
+  --(run-)freertos/prog PROG  Build/run a specific program part of the Demo. For example, this could be main_blinky,
+                        main_test_full, etc.
+  --(run-)freertos/bsp  BSP   This is only valid for the paramterized RISC-V-Generic. The BSP option chooses
+                        platform, RISC-V arch and RISC-V abi in the $platform-$arch-$abi format. See
+                        RISC-V-Generic/README for more details.
```

Comment:
This is auto-generated from --help so will disappear. Can you add it to the --help output instead?

Reply:
Fixed